src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,834 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect API to existing MySQL or PostgreSQL database',
+  description:
+    'Learn how to connect your API to an existing MySQL or PostgreSQL database.',
+    platforms: ['javascript', 'react', 'nextjs', 'angular', 'vue', 'react-native', 'flutter', 'swift', 'android']
+
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+<Callout warning>
+
+The following content requires you to deploy the Amplify GraphQL APIs via AWS Cloud Development Kit (CDK). If you have not yet deployed an Amplify GraphQL API with AWS CDK yet, review [Set up GraphQL API](/[platform]/build-a-backend/graphqlapi/set-up-graphql-api).
+
+</Callout>
+
+
+In this section, you'll learn how to:
```

Comment:
@chrisbonifacio In the future note that we are moving away from this format - this should be rewritten as an opening paragraph instead.